specifyweb/settings/specify_settings.py

```diff
@@ -30,6 +30,10 @@
 # Set to true if asset server requires auth token to get files.
 WEB_ATTACHMENT_REQUIRES_KEY_FOR_GET = False
 
+PUBLIC_IMAGE_SERVER_FILEUPLOAD_URL = None
```

Comment:
Maybe having a setting for the server base URL would be better so that it could be used for both getting and uploading? So like `PUBLIC_IMAGE_SERVER_BASE_URL = `. Then, your code can append `/upload_image` or `${filename}/...`

Reply:
Good idea!